crates/holochain/src/conductor/conductor.rs

```diff
@@ -2127,41 +2127,61 @@ mod service_impls {
             self.services.share_ref(|s| s.clone())
         }
 
-        pub(crate) async fn initialize_deepkey(
+        pub(crate) async fn initialize_services(self: Arc<Self>) -> ConductorResult<()> {
+            if let Some(installation) = self.get_state().await?.conductor_services.dpki {
+                self.services.share_mut(|s| {
+                    let dpki =
+                        DeepkeyBuiltin::new(self.clone(), self.keystore().clone(), installation);
```

Comment:
```suggestion
                    // TODO - at some point in the future this should be configurable instead of hard-coded.
                    let dpki =
                        DeepkeyBuiltin::new(self.clone(), self.keystore().clone(), installation);
```

Reply:
This isn't hard-coded, it's state that's set when the user installs Deepkey. Currently as an MVP there's no way to install a new version, since DPKI is considered a singleton and that would wreck all currently installed apps.